UsersGuide/source/parametersensitivity.rst

```diff
@@ -0,0 +1,101 @@
+Parameter Sensitivities with OpenModelica
+=========================================
+
+This section describes the use of OpenModelica to compute parameter
+sensitivities using forward sensitivity analysis together with
+sundials/idas solver.
+
+*Note: this is a very short preliminary description which soon will be
+considerably improved, since this a rather new feature and will
+continuous improved.*
+
+*Note: A OpenModelica version newer than openmodelica_1.10.0~dev-598-g33dd447
```

Comment:
I guess this should rather say: this feature was introduced in OpenModelica v1.10 (the development versions can be skipped for when features are added)